raisinbread/instruments/EEG_Acquisition_form.rules

```diff
@@ -0,0 +1,44 @@
+research_staff_two{-}Not Required{-}research_staff_two{@}=={@}NEVER_REQUIRED
```

Comment:
update these field names below after changes above...

Reply:
Not sure of this one, what changes are you refering to?